jrnl/plugins/yaml_exporter.py

```diff
@@ -80,9 +77,32 @@ def export_entry(cls, entry, to_multifile=True):
         dayone_attributes = ""
         if hasattr(entry, "uuid"):
             dayone_attributes += "uuid: " + entry.uuid + "\n"
-            # TODO: copy over pictures, if present
-            # source directory is  entry.journal.config['journal']
-            # output directory is...?
+        if (
```

Comment:
Sometime in the future, I'd love to split this off into a DayOne-specific class, so it's not clogging up the yaml exporter.